tracer/test/snapshots/OwinWebApi2Tests.NoFF.__path=_api2_delayAsync_0_statusCode=200.verified.txt

```diff
@@ -11,12 +11,16 @@
       aspnet.controller: conventions,
       aspnet.route: api2/{action}/{value},
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: api2/{action}/{value},
       http.status_code: 200,
-      http.url: http://localhost:00000/api2/delayasync/0,
+      http.url: http://localhost/api2/delayAsync/0,
+      http.useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36,
       http_useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36,
```

Comment:
Urgh, just noticed this, I guess we do need to keep `http_useragent` to we? 😭

Reply:
Actually, we don't need to keep this! We just happened to set `DD_HEADER_TAGS=User-Agent:http_useragent` in our OwinWebApi2Tests constructor. We should get rid of it in a separate PR though so it doesn't muddle up this PR

Reply:
PR: https://github.com/DataDog/dd-trace-dotnet/pull/2986